### PR TITLE
Yes no scheme

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Have added a yes no scheme which I intend to use across the 'yes_no_reason' radio questions. Pending https://github.com/AfricasVoices/Project-ICRAF/pull/2

### CODE REVIEW

**Comment by as2388** [comment]
@IsaackMwenda please merge master into this branch

**Comment by IsaackMwenda** [comment]
> @IsaackMwenda please merge master into this branch

@as2388  Merged PTAL.

File: code_schemes/yes_no.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+    "Version": "0.0.0.1",
+    "Name": "Yes_No",
```
Comment (as2388) [inline code_schemes/yes_no.json]: Which question is this a yes/no response to? Please include this in the scheme name.

**Review by as2388 — CHANGES_REQUESTED** [review]

File: code_schemes/yes_no.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/yes_no.json]: Have renamed to Radio Questions Yes_No. The scheme is shared across the Yes_No radio questions.

File: code_schemes/yes_no.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
+{
+    "Version": "0.0.0.1",
+    "Name": "Radio Questions Yes_No",
```
Comment (as2388) [inline code_schemes/yes_no.json]: Can we have a different yes_no scheme for each week? The yes/no labels are being used for different questions in each week. Having different schemes means we can't accidentally make non-sensical comparisons between weeks.

**Review by as2388 — CHANGES_REQUESTED** [review]

File: code_schemes/yes_no.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/yes_no.json]: Thanks! have created separate yes_no schemes for the respective Radio questions.

**Review by as2388 — APPROVED** [review]
Thanks @IsaackMwenda.

Please request review from @lukechurch and RDA

**Pull request merged**